uebung/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Phone-Throw</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css"/> 
+    <link href="https://fonts.googleapis.com/css2?family=Merriweather&display=swap" rel="stylesheet"> 
+</head>
+
+<body>
+    <header>
+        <!-- option of planet -->
+        <nav>
+            <ul>
+                <button type="button" id="earth"><li>Earth</li></button>
```

Comment:
eine liste besteht immer aus listen elementen
entweder du verzichtest auf das ul oder du packst die buttons noch in li's rein